apps/app/src/pages/Build/BuildHeader.tsx

```diff
@@ -9,15 +9,26 @@ import { FragmentType, graphql, useFragment } from "@/gql";
 import { BrandShield } from "@/ui/BrandShield";
 import { MagicTooltip } from "@/ui/Tooltip";
 
-const BrandLink = memo(() => {
-  return (
-    <MagicTooltip tooltip="Go to home">
-      <Link to="/" className="transition hover:brightness-125">
-        <BrandShield height={32} />
-      </Link>
-    </MagicTooltip>
-  );
-});
+const BrandLink = memo(
+  ({
+    ownerLogin,
+    repositoryName,
+  }: {
+    ownerLogin: string;
+    repositoryName: string;
+  }) => {
+    return (
+      <MagicTooltip tooltip="Go to home">
```

Comment:
This link already exists, it is called repository link. I don't understand why we change this now.

Reply:
Do we have a user that complains about this button?